### PR TITLE
Add io.github.mandruis7.xbox-cloud-gaming-electron

### PR DESCRIPTION
**Pull request by mandruis7**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.mandruis7.xbox-cloud-gaming-electron.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - org.freedesktop.Sdk.Extension.node16
+command: start-xbox-cloud-gaming-electron.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=system-bus
```
Comment (hfiguiere) [inline io.github.mandruis7.xbox-cloud-gaming-electron.yaml:17]: this is not allowed.

Comment (mandruis7) [reply io.github.mandruis7.xbox-cloud-gaming-electron.yaml:17]: I can potentially remove the --socket=system-bus just QA what effects has on the app

Comment (hfiguiere) [reply io.github.mandruis7.xbox-cloud-gaming-electron.yaml:17]: not *potentially*. Individual dbus name required should be added as needed.

Comment (mandruis7) [reply io.github.mandruis7.xbox-cloud-gaming-electron.yaml]: Just removed the line

**Comment by hfiguiere** [comment]
bot, build io.github.mandruis7.xbox-cloud-gaming-electron

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: that's disappointing.

Comment (mandruis7) [reply flathub.json:2]: I'm not totally close yet to have an ARM port but need to get a device first to test it out

**Comment by mandruis7** [comment]
bot, build io.github.mandruis7.xbox-cloud-gaming-electron

**Comment by barthalion** [comment]
```
    <summary>Open-source Xbox Cloud Electron client</summary>
    <description><p>A Linux desktop web app for Xbox Cloud Gaming</p></description>
```

It should have a disclaimer like at https://beta.flathub.org/apps/org.standardnotes.standardnotes about not being affiliated with Microsoft/Xbox.

**Comment by mandruis7** [comment]
> ```
>     <summary>Open-source Xbox Cloud Electron client</summary>
>     <description><p>A Linux desktop web app for Xbox Cloud Gaming</p></description>
> ```
> 
> It should have a disclaimer like at https://beta.flathub.org/apps/org.standardnotes.standardnotes about not being affiliated with Microsoft/Xbox.

This should have been added with the latest commit

**Pull request closed without merging**